Tasks/AzureResourceGroupDeployment/operations/ResourceGroup.ts

```diff
@@ -146,12 +146,21 @@ export class ResourceGroup {
         return depName;
     }
 
-    private updateOverrideParameters(parameters: Object): Object {
+    private updateOverrideParameters(template: Object, parameters: Object): Object {
         tl.debug("Overriding Parameters..");
 
         var override = parameterParser(this.taskParameters.overrideParameters);
         for (var key in override) {
             tl.debug("Overriding key: " + key);
+            try {
+                if (template["parameters"][key]["type"] == "int") {
```

Comment:
Are there more variable types which might require special behavior? Can you think through all the variable types and see that our behavior is correct